client/components/seo/facebook-preview/index.jsx

```diff
@@ -31,11 +33,7 @@ export class FacebookPreview extends PureComponent {
 		return (
 			<div className={ `facebook-preview facebook-preview__${ type }` }>
 				<div className="facebook-preview__content">
-					{ image && (
-						<div className="facebook-preview__image">
-							<img src={ image } />
-						</div>
-					) }
+					<div className="facebook-preview__image">{ image && <img src={ image } /> }</div>
```

Comment:
Should the conditional render be on the whole `div`, or do we need to maintain an empty `div` for some reason?

Reply:
So, this is a kind of effort of trying to avoid regressions.

The `img` is styled as `width: 100%; height: auto`, which means that it's as large as its container, and the height is calculated accordingly.

Though, that container is `width: 158px; height: 158px`, no matter what it contains or if it contains anything at all.

If we alternated between image and empty container, we would need to style both as `width: 158px; height: 158px;`, risking to screw up the image aspect ratio.